content/docs/insights/accounts.md

```diff
@@ -130,14 +130,49 @@ The AWS scanner for Pulumi Cloud requires access to the AWS account you want to
 
 ### Azure
 
-The Azure scanner for Pulumi Cloud requires access to your Azure account. This access can be granted by creating an ESC environment that, when opened, produces valid credentials to use the Pulumi Azure provider. Below are the steps to configure Azure credentials.
+The Azure scanner for Pulumi Cloud requires access to your Azure account. This access can be granted by creating an ESC environment that, when opened, produces valid credentials to use the Pulumi Azure provider.
+
+{{< notes type="info" >}}
+  We recommend using OpenID Connect (OIDC) for passwordless authentication with Azure. This method is more secure than using client secrets as it eliminates the need to store and manage long-lived credentials.
+{{< /notes >}}
+
+#### Option 1: OIDC authentication (recommended)
+
+1. Create a Microsoft Entra application and configure federated credentials:
+   * Follow the steps in [Configuring OpenID Connect for Azure](/docs/esc/environments/configuring-oidc/azure/)
+   * When configuring the federated credential:
+     * **Audience**: `azure:{your-org-name}`
+     * **Subject identifier**: `pulumi:environments:org:{your-org-name}:env:{project-name}/{environment-name}`
```

Comment:
```suggestion
     * **Subject identifier**: `pulumi:environments:org:<your-pulumi-org-name>:env:<esc-project-name>/<esc-environment-name>`
```

For greater clarity and consistency with our convention of using `<` and `>` as delimiters (such as it is).